pkg/controller/controller.go

```diff
@@ -65,8 +68,28 @@ func (c *Controller) Start(ctx context.Context) error {
 		return err
 	}
 
-	dnsInit := dns.NewDaemon(c.client)
-	go wait.UntilWithContext(ctx, dnsInit.RenewAndSync, dnsRenewPeriodHours)
+	go func() {
+		var success bool
+		for i := 0; i < 10; i++ {
+			// This will error until the cache is primed
+			if _, err := config.Get(ctx, c.Router.Backend()); err == nil {
+				success = true
+				break
+			} else {
+				time.Sleep(time.Millisecond * 100)
```

Comment:
This is a total of 1 second right? I’d do like 10 minutes. If it’s a huge cache it could take a long time to load.

Reply:
ah ok

Reply:
oh man, i meant for it to be 10 seconds total anyway, but yeah will do 10 mins